testbook/client.py

```diff
@@ -58,7 +58,11 @@ def execute_cell(self, cell, **kwargs):
 
         executed_cells = []
         for idx in cell_indexes:
-            cell = super().execute_cell(self.nb['cells'][idx], idx, **kwargs)
+            try:
+                cell = super().execute_cell(self.nb['cells'][idx], idx, **kwargs)
+            except Exception as e:
```

Comment:
This should be a CellExecuteError instead of generic Exception so that errors in nbclient itself still raise an appropriate stack trace